support-models/src/main/scala/com/gu/support/workers/states/SendAcquisitionEventState.scala

```diff
@@ -6,22 +6,37 @@ import com.gu.support.encoding.CustomCodecs._
 import com.gu.support.promotions.PromoCode
 import com.gu.support.redemptions.RedemptionData
 import com.gu.support.workers.{PaymentMethod, User, _}
+import io.circe.Decoder
 import org.joda.time.LocalDate
 
-case class SendAcquisitionEventState(
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+
+trait SendAcquisitionEventState extends FailableState {
+  def requestId: UUID
+  def giftRecipient: Option[GiftRecipient]
+  def product: ProductType
+  def paymentMethod: Either[PaymentMethod, RedemptionData]
+  def firstDeliveryDate: Option[LocalDate]
+  def promoCode: Option[PromoCode]
+  def acquisitionData: Option[AcquisitionData]
+}
```

Comment:
this trait forces the output of the previous, createZuora, step (namely the SendEmail state) to provide the correct fields for the acquisition state.
Unfortunately you can't inherit a case class and you can't deserialise a trait with the semiauto, so the trait is a duplicate of the Impl below.  However at least they are right beside each other.